### ISSUE TITLE
Error building on GHC 8.0.1

### ISSUE THREAD

**Issue by newhoggy** [bos/statistics#93]
```
Statistics/Correlation/Kendall.hs:11:1: error:
    parse error on input ‘-- $$\tau = \frac{n_c - n_d}{\sqrt{(n_0 - n_1)(n_0 - n_2)}}$$
--
-- where $n_0 = n(n-1)/2$, $n_1 = number of pairs tied for the first quantify$,’
builder for ‘/nix/store/cj4v6cki7jxikrzg3f4msx71b35993g8-statistics-0.13.2.3.drv’ failed with exit code 1
```

**Comment by dmcclean** [comment]
I am also experiencing this issue with the version of `statistics` that's included in stackage nightly.

**Comment by Shimuuar** [comment]
Thank you  for the report. Failure was due to haddock. It's fixed in bc6e3aebda4a3de01aebfab6c427a3fa3eed37a7. Will upload fix to hackage soon
